services/github/github-followers.service.js

```diff
@@ -26,57 +22,42 @@ module.exports = class GithubFollowers extends LegacyService {
     }
   }
 
+  static render({ followers }) {
+    return {
+      message: metric(followers),
+      color: '4183C4',
+      style: 'social',
+    }
+  }
+
   static get examples() {
     return [
       {
         title: 'GitHub followers',
         namedParams: {
           user: 'espadrine',
         },
-        staticPreview: {
+        staticPreview: Object.assign(this.render({ followers: 150 }), {
           label: 'Follow',
```

Comment:
Wasn't sure how badly we wanted to the example listing to keep the `Follow` label, but this was the only way I could get it to work

Reply:
Yea… I think I left a literal in one of the other GitHub services I refactored recently. This seems better.